src/connection_transfer/connection_transfer.go

```diff
@@ -15,10 +14,21 @@ const (
 	CONNECTION_TRANSFER_HANDLER_NAME = "connectionTransfer"
 )
 
+type ConnectionTransferConfig struct {
+	UpdateTopic string `toml:"update_topic"`
+}
+
 var (
+	connectionTransferConfig = ConnectionTransferConfig{
```

Comment:
instead of doing this for the configuration of every component, make the whole configuration be a singleton and provide a getter for the component configuration that panics if the configuration is not initialized